src/adaptors/http1/http1_server.c

```diff
@@ -1340,6 +1361,20 @@ static _server_request_t *_create_request_context(qdr_http1_connection_t *hconn,
     DEQ_INIT(hreq->responses);
     DEQ_INSERT_TAIL(hconn->requests, &hreq->base);
 
+    //
+    // Start a vanflow record for the server side request. The parent of this vanflow is
+    // its connection's vanflow record.
+    //
+    hreq->base.vflow = vflow_start_record(VFLOW_RECORD_FLOW, hconn->vflow);
+    vflow_set_uint64(hreq->base.vflow, VFLOW_ATTRIBUTE_OCTETS, 0);
+    vflow_add_rate(hreq->base.vflow, VFLOW_ATTRIBUTE_OCTETS, VFLOW_ATTRIBUTE_OCTET_RATE);
```

Comment:
Same as on the request.  Let's not add rate computation on the inner flows.

Reply:
Done ! Removed the rate computation calls on the inner flows.